packages/bratus-app/src/components/ComponentNode/ComponentNode.sc.js

```diff
@@ -32,6 +32,9 @@ export const StyledNode = styled.div`
   border: ${({ isHighlighted }) => {
     return isHighlighted ? '1.5px solid black' : '1px solid black';
   }};
+  box-shadow: ${({ isHighlighted }) => {
```

Comment:
This is a cool line of code!

Reply:
Thanks, this pattern is around 113 years old